### DISCUSSION TITLE
Any interest in TOML support?

### DISCUSSION THREAD

**Discussion by leetrout** [go-task/task#1161] [Polls]
TOML is starting to have a moment in Python-land with TOML coming into 3.11 standard lib and `pyproject.toml` being the new build system definition.

Any interest or willingness to revisit TOML support? I know it was removed in 2017 so maybe a non-starter...

**Comment by andreynering** [comment]
Hi @leetrout,

Thanks for the idea, but that will likely never happen.

The past experience proved that TOML is not a good fit for Task because it differs too much from YAML/JSON, and we really need its features (deep object indentation, etc).

Also, with regard to implementation, we're deeply tied to the YAML library we use, and adapt to something else wouldn't be trivial.

And last thing: the interest on JSON and specially TOML support was minimal at the time, and this was one of the strongest reasons to remove it.
